docs/best-practices/git-standardization.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Git Standardization
+---
+Document is going to contains the different standards relate to the use of Git within Mobile Jazz. 
+
+## Topics
+* [Commit messages](https://harmony.mobilejazz.com/docs/best-practices/git-standardization#commit-messages)
+* [Merge / Pull requests](https://harmony.mobilejazz.com/docs/best-practices/git-standardization#merge--pull-requests)
+* [Git methodology](https://harmony.mobilejazz.com/docs/best-practices/git-standardization#git-methodology)
+
+### Commit messages
+Everybody is free to use their own system in their git branches for commit messages but we highly recommend
+using [semantic commit messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716).
+
+### Merge / Pull Requests
+* Must be created for every feature / bug request associated with their asana task. If not asana task created, we must create it and notify the project manager.
+* Must be squashed using the Merge / Pull request title as the git commit message.
+* Must be rebased into the target branch.
+* Must use the follow title format described below.
```

Comment:
Must use the following title format described below
 or just
Must use the title format described below